### ISSUE TITLE
Start a Newsletter for local brigade

### ISSUE THREAD

**Issue by BretFisher** [Code4HR/tasks#35]
@nameispete and Anne are taking lead on sending out regular newsletters. After much discussion this is what was decided for now:

Also, sorry if there was an impression previously that I was :-1: on all this, but I’m a :+1: now that we've worked through some ideas and possibilities. Especially now that we've figured out how to reduce duplicate work for members. I'm excited.
### Tasking/process
- Use mailchip. No blog for now. We can create links to archive newsletters later on, but don't wait on that to start.
- c4hrva already has a mailchip account. I suggest we all use new individual accounts to help control access rather then sharing around a complex password. **I’ll invite you all.** It’ll let us see who last edited, who sent, and control access better.
- None of current 123 emails on that list are double-opt-in I think. We can just start sending newsletters to those people and hope they don't mark us spam :). Every email always has an unsubscribe link. Thoughts on this? **Should we send them an email upfront saying "you're now on the newsletter due to CA2014 involvement, click here to unsubscribe"?**
- [x] We’ll put the newsletter signup on the home page. Maybe right next to meetup button. We’ll talk to new members about it at hacknight. **I’ll look into this next week unless someone beats me to it.**
- [x] Add newsletter signup to meetup hacknight template so it's always mentioned. **I’ll look into this next week unless someone beats me to it.**
- [x] I'll mention the newsletter signup to the google group to get more of them signed up.
- [x] send newsletter invite to all meetup users week of Oct 5th
- [ ] Using automated workflow, it can be the **gateway drug to all the other signups and tasks we want to communicate to new members**. We can have emails kick off to users right at signup, another one X days later, etc… stepping them through a series of indoc steps. 
- [x] **Note that workflow feature requires paying $10 a month for Mailchip, which I will setup shortly.**
- Note these _workflow_ emails are based off their signup date. _Newsletter_ emails are unrelated but send to the same “List” in mailchip. One list, many emails. Hope this makes sense.
### Automated workflow individual email ideas:
- [ ] Upon newsletter signup we have a welcome email sent that tells them some basics (go signup for meetup, here’s our homepage, here's captain contacts in case they have questions)
- [ ] Next email automatically goes out to them 2 days later telling them about the brigade program (and signup at brigade.codeforamerica.org for national news)
- [ ] Next email auto sends another 3 days later telling them about google groups, IRC, and welcome.codeforhamptonroads.org, etc.
- [ ] ?

The more I think about this newsletter + automation aproach, the more I'm excited and glad @nameispete pushed the issue and got us all talking. People that likely want to be involved with mailchip are @kmcurry @wbprice @bcbarsness @stanzheng and a few others.

Oh and the +1/-1 stuff is based on Lazy Consensus "Consensus Building" [rules](http://openoffice.apache.org/docs/governance/consensusBuilding.html)

**Comment by bschoenfeld** [comment]
This is great. I'm definitely +1 on this and would be happy to contribute
when possible.

**Comment by BretFisher** [comment]
Shortcut to signup for newsletter: http://signup.c4hrva.us

@kmcurry do we still want all those categories? They are nice to have, but it makes the form a little more "intimidating". Also, when we add the signup to our homepage, it won't ask for anything but email, FYI.

**Comment by nameispete** [comment]
So, we have that "prototype" that Anne and I worked on. I'll try to figure out how to link it.

I see the newsletter as being a looked-forward-to digest of sorts. Obviously with news, but also "Call to Arms" asking for specific help from technical and non-technical readers. 

I am also interested to see if there is anybody out there who wants to do a "code-tip" as a feature below the fold. I've got a few ideas along these reader retention lines.

I would wait on setting up workflow for now until we can get each of the welcome letters polished. Also, I am not sure that if we send a new visitor four emails in the first week (1. welcome, 2. brigade, 3. groups, then 4, actual hack night digest) that we don't chase them away. Perhaps a slower rate of delivery might be appropriate. 

I also agree with Kevin that categories need revisiting. Before I saw this sign-up, I was thinking that we would list "interests" such as "Programming", "Usability Testing", "Writing", "Social Promotion (or yes, I have a Facebook account)", etc. But more along the lines-- what do YOU (the subscriber) want to be involved in. 

The mission of the newsletter might follow some guidelines:

Retain existing members by keeping them informed of events and opportunities within C4HRVA
Engage newcomers with a sense of "stuff happening"
Enlist help from members for promotion/time critical needs from projects

Lastly, and this is what I am playing with right now is the idea that we can use this medium to actually motivate people for the C4A agendas... okay so-- here's an example... Last night we talked about how the VA Health Dept doesn't have a scoring system, but we as an organization think that it should. Imagine in the newsletter (digest) we have an OpEd section where someone motivated writes a brief two/three paragraph article about this very topis with a prompt for action to "call your health department" We can then mirror content on tumblr... or wherever for posterity and if it's the right location, engage group discussion....

Also, if we announce future events like Start Norfolk ( @stanzheng -- nice picture on the home page btw), we may be able to get cross promotion from like-minded organizations. 

There's a lot in here and some of it is unfortunately lunch break ramblings, but I am excited to be behind this project and my heart is definitely into it. But while I was rush, rush, rush two weeks ago primarily to get buy-in and fellow champions, I am now reversing my position and want to make sure that we build this to last.

**Comment by TotoroMojo** [comment]
If anybody would like to send us projects in need of attention we would include them in a sort of call to arms section.  We will use this moving forward, so if a projcet is in need of beta testing or php development, etc.  let us know and we can include the project in the upcoming edition.

**Comment by kmcurry** [comment]
Put out a call to make a web video PSA for HRTB.us

On Thu, Sep 11, 2014 at 2:19 PM, Anne Tyrrell notifications@github.com
wrote:

> If anybody would like to send us projects in need of attention we would
> include them in a sort of call to arms section. We will use this moving
> forward, so if a projcet is in need of beta testing or php development,
> etc. let us know and we can include the project in the upcoming edition.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/c4hrva/tasks/issues/35#issuecomment-55305786.

## 

Kevin Curry
Virginia Beach, VA
http://www.linkedin.com/in/kevincurry

**Comment by kmcurry** [comment]
Typical Format:
- Feature Story
- Spotlight on a brigade member or project
- Spotlight on local relevant news item
- Spotlight on another brigade
- Upcoming events (ours and others')

**Comment by BretFisher** [comment]
As you can see at top of this issues checkboxes, the tasks are finally getting knocked out. Everyone here should have their mailchimp invite to write content, the signup link (http://signup.c4hrva.us) is now listed everywhere I could put it, and tonight I invited the Google Group to signup. We can do the same for meetup next week.

Tomorrow at 2pm an email will go out to the ~130 existing mailchimp addresses we already have from Code Across 2014 (they weren't opted in), asking them to signup for the newsletter list. @kmcurry: random thought, we could use that existing CA2014 list to also tell about DE Hackathon??

@venisebaxter has started playing with a new template. I know @TotoroMojo and @nameispete had one before, but I can't find it and we haven't seen them lately :(

Not sure how we should collect ongoing content ideas. For now I guess we just wing it in draft campaigns, but maybe eventually we could use these GitHub issues for helping with content.

I don't think any brigade captains need to give final approval (we're trying to get out of your way), so just think of us as resources to help you get what you need to send newsletters every ~2-4 weeks.

Lets ship the first one next Wednesday night!

**Comment by nameispete** [comment]
We do have a template. It’s unfortunately on a separate mailchimp account that I set up before getting access to the offiicial one. I am really psyched to hear that progress has been made without us. I think the newsletter is a very worthy idea and it's great to hear that the ball wasn't dropped simply because @TotoroMojo  and I were absent. 

Sorry that we’ve been absent We’ve had guests and work and some family issues that were pressing. We will see you all next week!

**Comment by BretFisher** [comment]
Peter we can still use your template!  In your mailchimp you can click down
arrow next to template to export, then in our account when we create a new
template choose “code you own” to import it.

Can you do that before next week so @venisebaxter can start putting her
content into it?

**Comment by nameispete** [comment]
[ It's crazy here... we moved back to the east coast to be closer to friends and family... I did not realize how many visiting people that entailed... ]

Here is the template: 

https://us9.admin.mailchimp.com/templates/share?id=32564606_5b5a804928402eea9d58_us9

You can import it directly into MailChimp just by using that link.

**Comment by BretFisher** [comment]
Great, thanks. @venisebaxter, here's their template in our account if you want to use it: https://us3.admin.mailchimp.com/templates/edit?id=120629

**Comment by bschoenfeld** [comment]
Where are we planning and organizing newsletter content? I just realized that the 29th is the one year anniversary of the bus app. By that time, we will have 150,000 page views, 35,000 sessions, and 6,000 users over our first year.

**Comment by kmcurry** [comment]
Could do a special edition for that.

On Thursday, October 9, 2014, Ben Schoenfeld notifications@github.com
wrote:

> Where are we planning and organizing newsletter content? I just realized
> that the 29th is the one year anniversary of the bus app. By that time, we
> will have 150,000 page views, 35,000 sessions, and 6,000 users over our
> first year.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/c4hrva/tasks/issues/35#issuecomment-58564594.

## 

Kevin Curry
Virginia Beach, VA

3Scape begins @ http://3Scape.me

**Comment by BretFisher** [comment]
Yea been thinking about the "collaborative content" problem of this newsletter and Mailchimp doesn't really solve that. We could do it here (new issue or this one) and mention @venisebaxter who's taking the lead on content atm. I'm happy to give a Mailcimp account to anyone who wants to add it in directly as well.

**Comment by venisebaxter** [comment]
I sent a newsletter, with content, to Kevin and Bret for approval. Check your emails!!

Venise Baxter 
Project Manager 
p. 757.351.8312
venise.baxter@homes.com

**Comment by kmcurry** [comment]
ship it!

On Thu, Oct 9, 2014 at 8:09 PM, venisebaxter notifications@github.com
wrote:

> I sent a newsletter, with content, to Kevin and Bret for approval. Check
> your emails!!
> 
> Venise Baxter
> Project Manager
> p. 757.351.8312
> venise.baxter@homes.com
> 
> > On Oct 9, 2014, at 8:03 PM, Bret Fisher notifications@github.com
> > wrote:
> > 
> > Yea been thinking about the "collaborative content" problem of this
> > newsletter and Mailchimp doesn't really solve that. We could do it here
> > (new issue or this one) and mention @venisebaxter who's taking the lead on
> > content atm. I'm happy to give a Mailcimp account to anyone who wants to
> > add it in directly as well.
> > 
> > —
> > Reply to this email directly or view it on GitHub.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/c4hrva/tasks/issues/35#issuecomment-58596782.

## 

Kevin Curry
Virginia Beach, VA

3Scape begins @ http://3Scape.me

**Comment by venisebaxter** [comment]
I can't! You two are the only ones with that permission!

Venise Baxter 
Project Manager 
p. 757.351.8312
venise.baxter@homes.com

**Comment by BretFisher** [comment]
I think it's great for a first release. _I'm looking it over now._  Main concern is about consistent branding. People never get our name right and the abbreviation is too hard for veterans to get right.

It Says "Code4HRVA" at top. Then mentions C4HRVA and Code for Hampton Roads later.

**Comment by BretFisher** [comment]
I just changed Code4HRVA and C4HRVA to Code4HR in the newsletter. This whole branding thing has been bugging me a while. Our new twitter is CodeforHRVA, github C4HRVA, facebook is codeforhamptonroads. 

Could we consolidate those all to one-single-name?  Why not Code4HR. Skip the VA, spell out the code.  Thoughts? (we don't need to decide this for shipping newsletter, but would like to improve our naming consistency).

**Comment by BretFisher** [comment]
We could use that same Code4HR name on twitter/facebook/domain/github/hashtag/ggroup/etc.

**Comment by kmcurry** [comment]
JFDI

On Thursday, October 9, 2014, Bret Fisher notifications@github.com wrote:

> We could use that same Code4HR name on
> twitter/facebook/domain/github/hashtag/ggroup/etc.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/c4hrva/tasks/issues/35#issuecomment-58599600.

## 

Kevin Curry
Virginia Beach, VA

3Scape begins @ http://3Scape.me

**Comment by wbprice** [comment]
Dig it. Do it.

**Comment by BretFisher** [comment]
OK newsletter decided it was best to send tomorrow at 4pm for "maximum engagement" so it's scheduled. Thanks ALL. I'll work on mailchimp permissions and the other branding later. Can someone change the twitter handle?

**Comment by wbprice** [comment]
@venisebaxter <- social media expert.  Says so on her LinkedIn.

**Comment by venisebaxter** [comment]
Working on it

Venise Baxter 
Project Manager 
p. 757.351.8312
venise.baxter@homes.com

**Comment by venisebaxter** [comment]
I only have access to the Twitter and Facebook, both are changed. 

Venise Baxter 
Project Manager 
p. 757.351.8312
venise.baxter@homes.com

**Comment by BretFisher** [comment]
Yea, one name to rule them all !! :100: !!
http://code4hr.org
http://meetup.com/code4hr
http://facebook.com/code4hr
http://twitter.com/code4hr
http://github.com/code4hr

Shortcuts:
http://docs.code4hr.org 
http://forum.code4hr.org 
http://meetup.code4hr.org 
http://signup.code4hr.org 
http://tasks.code4hr.org 
http://www.code4hr.org 

One issue I'll need to email everyone about, is their local git repo's will need the URL changed. Everything else should redirect to new locations. https://help.github.com/articles/changing-a-remote-s-url/

Also, landing page repo's are changing the name too.

**Comment by nameispete** [comment]
Like the change of Brand CODE4HR!! I like the newsletter!

**Comment by venisebaxter** [comment]
Yay, I'm glad to hear that

On Fri, Oct 17, 2014 at 1:25 AM, Peter Tyrrell notifications@github.com
wrote:

> Like the change of Brand CODE4HR!! I like the newsletter!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Code4HR/tasks/issues/35#issuecomment-59467858.

## 

Venise Baxter
_Project Manager_
p. 757.351.8312
venise.baxter@homes.com[image: Homes.com] http://www.homes.com/
[image: Homes Pro Facebook Fan Page]
https://www.facebook.com/HomesPro [image:
Homes Pro Twitter Page] http://twitter.com/#%21/HomesPro [image: Industry
trends from the HomesPro blog] http://connect.homes.com/category/blog/[image:
Connect with Homes.com on LinkedIn] http://www.linkedin.com/#%21/HomesPro

**Comment by qwo** [comment]
@venisebaxter knocked those out of the park!
